### PR TITLE
docs[Op#50983] Add system admin and user guides for OneDrive/Sharepoint

### PR DESCRIPTION
**Pull request by akabiru**
https://community.openproject.org/work_packages/50983

Define a **_placeholder_** for the new OneDrive/Sharepoint integration documentation- currently assigned to Op Product

### CODE REVIEW

File: docs/user-guide/file-management/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+keywords: files, attachment, Nextcloud, OneDrive, Sharepoint
+---
+
+# File Management
+
+| Topic                                                        | Content                                                      |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Manual upload](#manual-upload)                              | How to manually upload files to work packages in OpenProject. |
+| [Nextcloud integration](#nextcloud-integration)              | How to manage files using Nextcloud integration in OpenProject. |
+| [OneDrive/SharePoint integration](#onedrive/sharepoint-integration-enterprise-add-on) | How to manage files using OneDrive/Sharepoint integration in OpenProject. |
```
Comment (birthe) [inline docs/user-guide/file-management/README.md]: Does the "/" work in the title? @as-op ?

Comment (as-op) [reply docs/user-guide/file-management/README.md]: "/" does work in the title, but not in the #anchor link. It is just removed so please link to `#onedrivesharepoint-integration-enterprise-add-on`

**Comment by Kharonus** [comment]
Ahoi @as-op ,

I just updated this branch with my review from technical perspective. I had to alter some of the links and fix some previously moved ones. Can you check, if there are still some dead ones? Especially some of the TOC links in some readmes are displayed as not reachable in my IDE.

**Review by as-op — APPROVED** [review]

**Comment by as-op** [comment]
Hi @Kharonus The linkchecker catched them all. I fixed them.

**Pull request merged**